catalog/spatial/registry/registry-federation-admin-service-impl/src/test/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitializationTest.java

```diff
@@ -143,66 +142,11 @@ public void initWithPreviousEntryWithNameChange() throws Exception {
     identityNodeInitialization.init();
 
     assertThat(updatedMetacard.getValue().getTitle(), is(CHANGED_TEST_SITE_NAME));
+    assertThat(updatedMetacard.getValue().getId(), is(notNullValue()));
     verify(federationAdminService, times(1)).updateRegistryEntry(any(Metacard.class));
     verify(federationAdminService, never()).addRegistryEntry(any(Metacard.class));
   }
 
-  @Test
-  public void initWithGetLocalRegistryNodeException() throws Exception {
```

Comment:
❓ just curious why all these tests got removed?

Reply:
They were testing failure count or initialization failure. The failures now don't bubble up as an exception we can test for. I added some of the tests back in with some other validation checks.